src/main/webapp/resources/css/inward_five_five.css

```
@@ -0,0 +1,365 @@
+    font-weight: bold;
+}
+
+.itemsBlock{
+    text-align: center!important;
+    font-size: 13px!important;
+    font-family:verdana, Helvetica, sans-serif!important;
+    text-transform: capitalize!important;
+}
+
+
+itemsBlockRight{
```

Stylelint (17.14.0)

[error] Expected "itemsBlockRight" to be "itemsblockright" (selector-type-case)

(selector-type-case)